content/get-started/docker-concepts/building-images/multi-stage-builds.md

```diff
@@ -233,9 +233,14 @@ Now that you have the project, you’re ready to create the `Dockerfile`.
     ```console
     $ docker run -d -p 8080:8080 spring-helloworld
```

Comment:
Thanks for catching this error. Actually, I think I would prefer that we rather drop the `-d` flag to have the container logs print directly to the terminal, avoiding a background container and log inspection altogether.

```suggestion
    $ docker run -p 8080:8080 spring-helloworld
```